data/421/177/883/421177883.geojson

```diff
@@ -86,7 +86,7 @@
         }
     ],
     "wof:id":421177883,
-    "wof:lastmodified":1627522351,
+    "wof:lastmodified":1628127588,
```

Comment:
If no property or geom changes, shouldn't be included in the diff.

Reply:
I've adjusted the script I'm using to guard against this in future commits